data/projects/implement_aspects.md

```diff
@@ -1,29 +1,63 @@
+Initial configuration of coala is **hard**, especially for new user. There
+are few reason for this. First is because new user is not always familiar with
+all the functionality provided by various tools inside coala. Beside that, coala
+tries to provide a full set of option for those tools. And also there is a
+possibility that the same type of checking is executed by different tools to
+the same line of code, resulting in duplicated report.
+
 In the future, coala will choose the bears to run for the user based on what
 aspects of a programming language he wants to analyze. This project should
 realize this.
 
```

Comment:
Intro sentence to connect:"This is where aspects do their work. They are categories of analysis routines, for example code clone detection.
Right now, a user would have to find a tool that does code clone detection for his language and configure it through coala. With aspects, the user just adds the Code-Clone-Detection aspect and coala figures out if there is a bear that supports that analysis for the given language and also set useful default parameters. This takes the burden off the user to know what tool he needs. Instead they only have to know what they want to check for."

You don't have to copy paste this, just an idea for the style of storrytelling here.